azure-pipelines/ci.yml

```diff
@@ -0,0 +1,57 @@
+trigger:
```

Comment:
you can directly use azure-pipelines.yml in root folder level instead of putting it into a folder.

Reply:
to test pipeline

Reply:
I want to create a release pipeline later to separate it

Reply:
For release pipeline. It's private you can't put it into here.

Reply:
I mean in a separate file but in azure-pipelines folder. That's why I create a folder. And the release pipeline will sure be put in internal pipelines.

Reply:
Release pipeline YAML  file can't be put into GitHub repo.   It is stored in DevOps. So this repo only contains one pipeline. Create a folder will make it no clear.

Reply:
changed